install-dev/index.php

```diff
@@ -25,8 +25,8 @@
  */
 
 
-if (!extension_loaded('SimpleXML')) {
-    header('Location: missing_extension.html');
+if (!extension_loaded('SimpleXML') || PHP_VERSION_ID < 50400) {
+    require_once __DIR__.'missing_extension.php';
```

Comment:
` __DIR__` was introduced with PHP 5.3. 
With PHP 5.2, this `require_once` will break.